src/Components/Navbar.jsx

```diff
@@ -74,6 +99,7 @@ const Navbar = () => {
         <option value="">Delhi</option>
```

Comment:
for hard-coded strings used constants. Make a separate file add all string there.